runtime/index.d.ts

```diff
@@ -39,15 +94,26 @@ declare module '@sapper/common' {
 		redirect: (statusCode: number, location: string) => void;
 	}
 
-	export interface Page {
+	export type PageParams = Record<string, string>;
+	export type Query = Record<string, string | string[]>;
+	
+	export interface PageContext {
```

Comment:
I'd probably rather save the nice name `Page` for external usage and rename the internal `Page` type to something like `PageInfo`

Reply:
I'd be fine with that, but I generally feel this *isn't* a "page". The page parameters really parametrize a route, so I guess a page is more or less the same as a route, This object is attached to a request, whereas a page feels like something more permanent, at least to me.

What about `PageRequest` (a request for a page)? The request carries the parameters for the page, the `PageParams`. That kind of makes sense.

Reply:
It's not really a `PageRequest` because of the `error`. I guess `PageContext` makes sense